modular_zubbers/code/modules/limbgrowncorpses/limbgrowerblanks.dm

```diff
@@ -0,0 +1,31 @@
+// Used for printing dead bodies with the limbgrower
+/mob/living/carbon/human/empty
+
+/mob/living/carbon/human/empty/Initialize(mapload)
+	. = ..()
+	var/old_death_sound = death_sound // You can't do this with initial() as the death sound is set after by the species datum
+	death_sound = null // We don't need them screaming
+	death()
+
+	var/obj/item/organ/internal/brain/B = get_organ_slot(ORGAN_SLOT_BRAIN)
+	if(B)
+		B.Remove(src)
+		QDEL_NULL(B)
+
+	death_sound = old_death_sound // Now they can die loudly again
```

Comment:
Normal humans don't have death sounds, also the nullcheck is not needed as the brain is guaranteed to exist and qdelling already removes the organ from the body so we don't need to do that.

Turns out my old synth code that you copied is not that good lol

```suggestion
	death()

	var/obj/item/organ/internal/brain/to_remove = get_organ_slot(ORGAN_SLOT_BRAIN)
	QDEL_NULL(to_remove)
```